hotel-management/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "counter-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "eslint-import-resolver-typescript": "^3.6.1",
```

Comment:
Please recheck which one should belong to dependencies or devDependencies?

Reply:
This is a mistake, eslint should've been moved to devDependencies, I will fix it.